src/mongocrypt-marking.c

```diff
@@ -235,24 +237,27 @@ void _mongocrypt_marking_cleanup(_mongocrypt_marking_t *marking) {
             return false;                                                                                              \
         }                                                                                                              \
                                                                                                                        \
-        if (!useCounter) {                                                                                             \
+        if (!useContentionFactor) {                                                                                    \
             /* FindEqualityPayload uses *fromDataToken */                                                              \
             _mongocrypt_buffer_copy_to(mc_##Name##DerivedFromDataToken_get(fromDataToken), out);                       \
             mc_##Name##DerivedFromDataToken_destroy(fromDataToken);                                                    \
             return true;                                                                                               \
         }                                                                                                              \
                                                                                                                        \
-        BSON_ASSERT(counter >= 0);                                                                                     \
+        BSON_ASSERT(contentionFactor >= 0);                                                                            \
         /* InsertUpdatePayload continues through *fromDataTokenAndCounter */                                           \
```

Comment:
```suggestion
        /* InsertUpdatePayload continues through *fromDataTokenAndContentionFactor */                                           \
```

Reply:
Right. Plus all the local vars in this block. Good catch. Done and done.